external/miniVite/README.md

```diff
@@ -13,91 +12,100 @@ The miniVite version that works with Metall comes with a CMake file.
 On this instruction page, we use the CMake file to build miniVite with UMap.
 
 
+# Build
 ## Required
+
 We assume that the following items are already available (installed) on the system:
 - GCC 8.1 or more.
 - MPI
 - CMake 3.10 or more.
 
+The write-protect feature of userfaultfd() must be available on the system's Linux kernel.
 
-## Build with Spack
+## Build Manually
+
+miniVite uses header files of Boost C++ Libraries and Metall. One does not needs to build them.
+On the other hand, one has to build and install UMap.
+To install UMap, see [here](https://github.com/LLNL/umap).
+
+Without Spack, one has to tell miniVite's CMake file the location of required libraries.
+Here are the CMake variables to specify the locations of Boost C++ Libraries, Metall, and UMap manually.
```

Comment:
No point in mentioning Spack here since it is only referenced later on. Maybe revise to:
Set locations of required libraries for miniVite's CMake to use.

Reply:
Good point. Fixed it.